package.json

```diff
@@ -21,7 +21,8 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "serve": "vite preview",
-    "lint": "eslint --ext .jsx,.js,.ts,.tsx --fix .",
+    "lint": "eslint \"src/**/*.{ts,tsx,js}\"",
```

Comment:
Why do you exclude all files which not from `src`? I thought we agreed to lint config files too